tests/bluetooth/tester/testcase.yaml

```diff
@@ -16,6 +16,20 @@ tests:
       - frdm_rw612
     extra_configs:
       - CONFIG_NXP_MONOLITHIC_NBU=n
+  bluetooth.general.tester_hci_ipc:
+    build_only: true
+    harness: bluetooth
+    platform_allow:
+      - qemu_x86
+      - native_sim
```

Comment:
Do these two really need to be listed for this?

Reply:
No not really, but they will work so I kept them as allowed. The same argument could be had for the le_audio config so I tried to keep it similar. I thought CI only ran the `integration_platforms` anyway, or am I mistaken? I'm not very partial about this, I can remove them if you think it's better.